cfme/tests/storage/test_volume.py

```diff
@@ -224,7 +245,17 @@ def test_storage_volume_attached_delete():
             4. check for flash message " Cloud Volume "Volume_name" cannot be
             removed because it is attached to one or more Instances "
         """
-    pass
+    try:
+        attached_volume.delete(from_manager=from_manager)
+        pytest.fail("Attached volume was deleted!")
+    except Exception:
+        if from_manager:
+            view = attached_volume.browser.create_view(StorageManagerVolumeAllView,
```

Comment:
Note this doesn't actually matter, because you aren't asserting the view is displayed.  Either view could be used for the flash message assertion.